server/settings/tools.py

```diff
@@ -484,6 +484,17 @@ class GlobalToolsModel(BaseSettingsModel):
                 "task_types": [],
                 "tasks": [],
                 "template": "{folder[name]}_{variant}"
+            },
+            {
+                "product_types": [
+                    "textureSet"
+                ],
+                "hosts": [
+                    "substancedesigner"
+                ],
+                "task_types": [],
+                "tasks": [],
+                "template": "T_{folder[name]}{variant}"
```

Comment:
This feels overly specific to unreal - and I have my doubts about whether this is a sensible default?

Reply:
We can remove this here, and if the users want to add the template, we can suggest them to add the parameter accordingly to this.